src/dlgProfilePreferences.cpp

```diff
@@ -1779,3 +1804,37 @@ void dlgProfilePreferences::slot_script_selected(int index)
         preview->setText(mpHost->getActionUnit()->getAction(itemId)->getScript());
     }
 }
+
+/*!
+ * \brief dlgProfilePreferences::slot_changeShowSpacesAndTabs
```

Comment:
Yeah for sure, but everyone picking a different documentation system and just running with it won't work - as you've noticed, there's at least two - QDoc and Doxygen. That's why I mention we should all sit down together and figure out the one we want to use.